src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -149,14 +167,94 @@ export function LoadingTokenTable() {
   )
 }
 
-export default function TokenTable({ data }: { data: TokenData[] | undefined }) {
+interface TokenRowProps {
+  data: TokenData[]
+  index: number
+  style: CSSProperties
+}
+
+export default function TokenTable() {
   const showFavorites = useAtomValue<boolean>(showFavoritesAtom)
   const timePeriod = useAtomValue<TimePeriod>(filterTimeAtom)
-  const filteredTokens = useFilteredTokens(data)
-  const sortedFilteredTokens = useSortedTokens(filteredTokens)
+
+  const [page, setPage] = useState<number>(1)
+  const [topTokens, setTopTokens] = useState<TokenData[]>(
+    Array.from({ length: MAX_TOKENS_TO_LOAD }).map((_) => ({ loading: true } as TokenData))
```

Comment:
can we use this for the `LOADING_ROWS` piece above?

Reply:
not relevant anymore, changed loading table code.